packages/server-wallet/src/protocols/__test__/defunder.test.ts

```diff
@@ -26,6 +27,30 @@ const testLedgerChannel = TestChannel.create({
 
 let store: Store;
 
+function ensureCloseObjective(
+  channel: TestChannel,
+  participantIndex = 0
+): Promise<DBCloseChannelObjective> {
+  // add the closeChannel objective and approve
+  return store.transaction(async tx => {
+    const o = await store.ensureObjective(
+      channel.closeChannelObjective([participantIndex, Math.abs(participantIndex - 1)]),
```

Comment:
This is always equivalent to this, right?
```suggestion
      channel.closeChannelObjective([1, 0]),
```

Reply:
What if `participantIndex` is `0`. Then `txSubmitterOrder` is `[0, 1]`.

Reply:
... right

it's always equivalent to 
```suggestion
      channel.closeChannelObjective([participantIndex, 1 - participantIndex]),
```

which was going to be my suggestion until I confused myself!

Reply:
Good point c951cef7f